js/src/behaviors/dismiss.ts

```diff
@@ -0,0 +1,41 @@
+export function initDismiss() {
+	window.addEventListener('click', ({ target }: Event) => {
+		const dismissButton = target instanceof Element && target.closest('[data-dismiss]');
+
+		if (!dismissButton) {
+			return;
+		}
+
+		const dismissTarget = dismissButton.closest('[data-dismissable]');
+
+		if (!dismissTarget) {
+			return;
+		}
+
+		dismissElement(dismissTarget);
+	});
+}
+
+function dismissElement(dismissTarget: Element) {
+	const dismissAnimation = dismissTarget.getAttribute('data-dismiss-animation');
+
+	if (dismissAnimation) {
+		dismissTarget.addEventListener('animationend', () => {
```

Comment:
I don't think that's something needed right now. All possible values are locked in and are .4s. Let's get back to it if it will ever becomes a problem.